### PR TITLE
Completed Date Save Bug

### LINKED ISSUE #129: Bug - [User] Book Appointment in date invalid
**Describe the bug**

- when user book appointment on specific date it not able to store that date and time in database. Database save wrong value.

Please check in first priority. 

### Check List

- [x] I have linked my PR with this task
- [x] I have added comment after completing this task
- [x] I have followed all [guidelines](https://github.com/jaykishan-sewak/doctor-appoint-app/tree/main) for code

### PR DESCRIPTION
**Pull request by HarshPanchalInfusion**
PR For invalidate date

### CODE REVIEW

File: app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt [hunk 1]
```diff
@@ -98,7 +106,13 @@ class BookAppointmentFragment :
                     setMessage(resources.getString(R.string.dialog_appointment_desc))
                     neutralButton { dialog ->
                         dialog.dismiss()
-                        viewModel.addBookingAppointmentData(selectedTime)
+                        try {
+                            dateFormat = SimpleDateFormat(BOOKING_DATE_FORMAT)
+                            val combinedStr = "$dateStr $timeStr"
```
Comment (vikas-infusion) [inline app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt]: dateFormat.parse("$dateStr $timeStr") don't take extra variable.

File: app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt [hunk 2]
```diff
@@ -114,12 +128,34 @@ class BookAppointmentFragment :
         }
     }
 
+    private fun setDateAndTime(
```
Comment (vikas-infusion) [inline app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt]: Remove this setDateAndTime method i think unused method.

File: app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt (hunk 1 above)
Comment (HarshPanchalInfusion) [reply app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt]: Done

File: app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt (hunk 2 above)
Comment (HarshPanchalInfusion) [reply app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt]: Done

**Review by vikas-infusion — APPROVED** [review]

**Pull request merged**